### DISCUSSION TITLE
Seeing things I do not know at all

### DISCUSSION THREAD

**Discussion by OndrejSpanel** [community/community#16977] [Other Feature Feedback, Questions, & Ideas]
Often in my "For you (beta)" feed I see thinks which I am not interested in, in fact I have no idea what they are about.

Can this be finetuned somehow?

Why do I see https://github.com/keycloak/keycloak news? I am not aware ever visiting that repository and it does not seem to be related to topics I usualy visit.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
